src/App.js

```diff
@@ -14,11 +16,56 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const preparedGoods = goodsFromServer.map((good, index) => (
+  {
+    name: good,
+    id: index,
+  }
+));
+
+class App extends React.Component {
+  state = {
+    item: [],
```

Comment:
З мультиселектом там точно не один елемент буде
```suggestion
    selectedGoods: [],
```